config/main.py

```diff
@@ -788,6 +798,70 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN', vlan_name, vlan)
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
+@vlan.group('dhcp_servers')
+@click.pass_context
+def vlan_dhcp_servers(ctx):
+    pass
+
+@vlan_dhcp_servers.command('add')
+@click.argument('vid', metavar='<vid>', required=True, type=int)
+@click.argument('dhcp_server_ip', metavar='<dhcp_server_ip>', required=True, type=str)
+@click.pass_context
+def add_vlan_dhcp_servers(ctx, vid, dhcp_server_ip):
+    """ Add a DHCP server IP helper under the vlan """
+    if not is_ipaddress(dhcp_server_ip):
+        click.echo('Invalid ip address')
+        return
```

Comment:
Since you passed the Click context to this command, you can replace `click.echo()` with `ctx.fail()` and remove the `return` line. `ctx.fail()` will print and message and abort. There are other examples in this file.